src/main/java/org/sagebionetworks/bridge/sdk/exceptions/PublishedSurveyException.java

```diff
@@ -2,28 +2,25 @@
 
 import org.joda.time.DateTime;
 import org.joda.time.format.ISODateTimeFormat;
+import org.sagebionetworks.bridge.sdk.models.schedules.SurveyReference;
 
 public class PublishedSurveyException extends BridgeServerException {
 
     private static final long serialVersionUID = 5150621818895986256L;
```

Comment:
This is exactly why I don't like serialVersionUIDs. You changed the internal structure of a class, but missed updating the serialVersionUID. You should delete the serialVersionUID, suppress the warning, and just let the compiler auto-generate it. (This matters more for client SDK code, since the client might actually try to serialize these things.)